lib/serialization/specPath.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export class SpecPath {
```

Comment:
missing documentation for SpecPath that describes its purpose. Will be helpful for future developers.

Reply:
Good point. I'll add that.